src/Test/Unit/Config/Schema/_files/.magento.env.md

```diff
@@ -0,0 +1,20 @@
+# ENV_VARIABLE
+
+Some description
+
+|Attribute|Values|
+|---|---|
+|Type|string|
+|Magento version|\>=2.1.24|
+|Stages|global|
+
+## Examples
```

Comment:
Subsequently, this would be a heading 3.

Reply:
Fixed